expression/integration_test.go

```diff
@@ -7656,3 +7656,18 @@ func (s *testIntegrationSerialSuite) TestIssue20608(c *C) {
 	tk := testkit.NewTestKit(c, s.store)
 	tk.MustQuery("select '䇇Հ' collate utf8mb4_bin like '___Հ';").Check(testkit.Rows("0"))
 }
+
+func (s *testIntegrationSuite) TestVitessHash(c *C) {
+	defer s.cleanEnv(c)
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	tk.MustQuery("select vitess_hash(30375298039) from t").Check(testkit.Rows("\x03\x12\x65\x66\x1E\x5F\x11\x33"))
```

Comment:
Should we create a table `t` first?

Reply:
I did not create `t` for the reason that there are many other function tests that do not do that. For example tests for `str_to_date`, `hour`, `time`, etc. If you think it's important - I will add `t`.

Reply:
I think it's better to add some statements like
```
tk.MustExec("drop table if exists t;")
tk.MustExec("create table t(a int);")
```
for safty.